### DISCUSSION TITLE
How to Encourage Learners to Generate Project Ideas?

### DISCUSSION THREAD

**Discussion by myquite** [Vets-Who-Code/course-development-guide#7] [Ideas]
I want to discuss how best to teach our troops to continuously generate project ideas so they can whittle the list down to ideas they can test. The goal is to cultivate a mindset that encourages creative problem-solving and teaches the troops continuous discovery habits and product development skills.

**Comment by axecopfire** [comment]
How would you propose to teach this skill

**Reply by myquite** [reply]
I don't have any concrete ideas just yet. I see where people get stuck on what to build, and I want to help them work through that. It may be less of an issue than I think.
